k2/csrc/ragged_ops.h

```diff
@@ -1321,6 +1331,25 @@ template <typename T>
 Array1<T> ComputeHash(Ragged<int32_t> &src);
 
 
+/*
+  If `src` has two axes, this will return the unique sub-lists (in a possibly
+  different order, but without repeats).  If `src` has 3 axes, it will
+  do the above but separately for each index on axis 0; if more than 3 axes,
+  the earliest axes will be ignored.
+
+     @param [in] src  Source ragged tensor
+
+     @return   Returns a tensor with the same number of axes as `src` and
+            possibly fewer elements due to removing repeated sequences on the
+            last axis (and with the last-but-one indexes possibly in a different
+            order).
+
+  CAUTION: the current implementation does not completely remove the possibility
```

Comment:
If several sequences have the same hash, the current implementation
keeps only one of them.

Repeated sequences would have an identical hash value. Why does it say
> the current implementation does not completely remove the possibility of repeated sequnces,

Reply:
mm that was a mistake, it does not completely guarantee that all unique sequences will 
be present in the output.  Perhaps you can fix..

Reply:
Will fix it.